spec/ruby-push-notifications/mpns/mpns_connection_spec.rb

```diff
@@ -0,0 +1,46 @@
+
+module RubyPushNotifications
+  module MPNS
+    describe MPNSConnection do
+
+      describe '::post' do
+
+        let(:body) { 'abc' }
+        let(:headers) {
+          {
+            'x-notificationstatus' => ['Received'],
```

Comment:
True, you're right, but I don't like the shape of this, do you? 
It would be good to work the headers into a true hash so that outside the MPNSConnection Class nobody needs to know about that trick, makes sense? Sorry about that, but is quite tricky and error prone.